### PR TITLE
Feature/nd174 secrets accessible in plain text

### PR DESCRIPTION
**Pull request by asifamirat00**
ND-174 secrets accessible in plain text

### CODE REVIEW

**Review by juddin927 — APPROVED** [review]
LGTM

**Review by jamesgreen-moj — APPROVED** [review]
LGTM

File: modules/admin/iam.tf [hunk 1]
```diff
@@ -61,8 +61,13 @@ data "aws_iam_policy_document" "assume_role_policy" {
 }
 
 resource "aws_iam_role_policy_attachment" "ecs_task_execution_policy_attachment" {
+  for_each = toset([
+    "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy",
+    "arn:aws:iam::aws:policy/AmazonSSMFullAccess"
```
Comment (jamesgreen-moj) [inline modules/admin/iam.tf:66]: This should not be full access

**Review by jamesgreen-moj — CHANGES_REQUESTED** [review]

File: modules/admin/iam.tf (hunk 1 above)
Comment (asifamirat00) [reply modules/admin/iam.tf]: Thanks for reviewing James, recommendations taken on board. Now using the AmazonSSMReadOnlyAccess policy for read only access to SSM parameter store

**Review by jamesgreen-moj — APPROVED** [review]
LGTM

**Pull request merged**